### ISSUE TITLE
author property must be an object

### ISSUE THREAD

**Issue by aaronpk** [jackjamieson2/yarns-microsub-server#75]
It looks like the `author` property of posts is just a string. It needs to be a full `card` object so that apps can render the author better. If you only have a name, then just only include that value.

```json
{
  "type": "entry",
  ...
  "author": {
    "type": "card",
    "name": "Author Name",
    "url": "http://example.com",
    "photo": "http://example.com/photo.jpg"
  }
}
```

https://indieweb.org/Microsub-spec#card

**Comment by jackjamieson2** [comment]
Pretty sure that author is always returned as an object from Yarns, but maybe I'm missing cases where only one property is present.  I'll do some testing to double-check

**Comment by aaronpk** [comment]
I could be wrong but I assumed this screenshot was from Yarns: https://github.com/aaronpk/Monocle/issues/40#issue-448696810

![](https://user-images.githubusercontent.com/1940855/58401351-d8da2680-805d-11e9-978c-303ce7b26a97.png)

I noticed that it included `_permalink` which is not documented in the spec, but I found that property in the Yarns source code.

**Comment by jackjamieson2** [comment]
Hmm, maybe you're right. I have some parsing issues to fix with RSS feeds (#60).  Will make this a priority for this week

@dshanske, If you're working on feeds in Parse-This, this issue might overlap what you're already doing.  Either way I'll investigate and let you know if I think of ways to address this in Parse-This

**Comment by jackjamieson2** [comment]
I'm not able to replicate this problem, but I still wrote a function to ensure that author is always an object. Pushed to dev and I'll close this issue once I merge to the master branch
